src/data/validation/email.js

```diff
@@ -50,6 +55,11 @@ const validateEmailTemplateFields = (formData, templateKey, isSubjectRequired =
   return errorsDict;
 };
 
+const sanitizeEmail = (email) => {
+  // Strip bidirectional characters. This is mostly to make RTL emails compatible
```

Comment:
nit: perhaps amend the comment to include a mention that this only strips _invisible_ characters.